css/layout.css

```diff
@@ -32,3 +44,43 @@
     width: 50%;
   }
 }
```

Comment:
You media query should _always_ be the last things in your CSS file. Move this section below the extra CSS you wrote